cluster/test-network.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Copyright 2014 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${KUBE_ROOT}/cluster/kube-env.sh"
+source "${KUBE_ROOT}/cluster/${KUBERNETES_PROVIDER}/util.sh"
+
+${KUBE_ROOT}/hack/e2e-suite/goe2e.sh -tTestNetwork
```

Comment:
Is this syntax right? With POSIX flags, I'd expect `-tTestNetwork` to mean `-t -T -e -s -t -N ...`, so I think you want `-t TestNetwork` instead... Does pflag really work with this syntax?

Furthermore, `-t` is currently not working, I broke it in the Ginkgo conversion, though it's the next thing on my plate to fix... I expect to have a PR to fix it real soon now.